R/postProcessTo.R

```diff
@@ -182,7 +182,19 @@ postProcessTo <- function(from, to,
     if (is.null(projectTo)) projectTo <- to
   }
 
-  if (!all(is.null(to), is.null(cropTo), is.null(maskTo), is.null(projectTo), is.null(writeTo))) {
+  needPostProcessing <- !all(is.null(to), is.null(cropTo), is.null(maskTo),
+                             is.null(projectTo), is.null(writeTo))
+  if (needPostProcessing) {
+    if (.isGridded(from)) {
+      if (getOption("reproducible.leaveOnDisk", TRUE)) {
+        co <- capture.output(origMemFrac <- terra::terraOptions()$memfrac)
+        if (identical(origMemFrac, 0.5)) {
```

Comment:
so if the user has changed `memfrac` away from the default (up or down), this will always force memfrac 0? and this is enabled by default? this will likely become very confusing for users to manually manage both options.

Reply:
I believe the logic says the opposite, no? If the user hasn't changed it from the default, then leave on Disk. If they have set it to a different value, it means they know what they are doing and like it. All advanced users who want the objects to be pulled into memory, and have NOT changed the default `memfrac`, then they can set `leaveOnDisk = FALSE`